src/newton1d.jl

```diff
@@ -0,0 +1,74 @@
+function newton1d{T}(f::Function, f_prime::Function, x::Interval{T};
+                    epsX=eps(T), epsf=eps(T), debug=false)
+
+    L = Interval[]
+    R = Root[]
+    z = zero(x.lo)
+    push!(L, x)
+    while !isempty(L)
```

Comment:
Please add blank lines to separate pieces of code, and/or split up big functions into sub-functions.